src/components/Modal/ModalExample.tsx

```diff
@@ -84,7 +84,7 @@ export const ModalExample = ({
           </TextPassage>
         </ModalBody>
         <ModalFooter>
-          <ButtonGroup align="right">
+          <ButtonGroup>
```

Comment:
Didn't seem to make any change (left is `next`, right is this feature branch local):
![image](https://user-images.githubusercontent.com/86632227/162859472-e54b0c79-1844-4b2c-b00a-99782dc06896.png)

Reply:
I think theirs was broken. I would leave the `align="right"` and celebrate that the chromatic test will show it's now working.

Reply:
I am down to keep not having `align` prop. It's a matter of adding `width: 100%` and `justify-content: end`, but since the `ButtonGroup` itself is a wrapping `div`, the parent can style it as such.